content/06.methods.md

```diff
@@ -288,7 +288,8 @@ We plotted scores by cancer group using the `ComplexHeatmap` R package (**Figure
 ##### Transcriptomic Dimension Reduction (`transcriptomic-dimension-reduction` analysis module)
 
 We applied Uniform Manifold Approximation and Projection (UMAP) [@https://doi.org/10.48550/arXiv.1802.03426] to log2-transformed FPKM data for stranded RNA-Seq samples using the `umap` R package (See **Key Resources Table**).
-We set the number of neighbors to 15.
+We considered all stranded RNA-Seq samples for this analysis, but we removed any genes with fewer than a 100 total count across all such samples.
```

Comment:
It's not really a count in that case then right? (Despite what the parameter name, etc. says.) It would be genes with where the sum of the FPKM across samples was less than 100, I believe.

Reply:
Will rephrase!